backend/GymFeatures/GymAPI.py

```diff
@@ -105,3 +106,15 @@ def add_gymIsOpen():
     except:
         return {"err": "An error has occurred", "status": "failed"}, 500
     return make_response(response)
+
+@gym_api.route("/picture/profile/<string:userID>", methods=['GET'])
+def getUserPicture(userID):
+    try:
+        profile = db.Profiles.find_one(
+            {"userID": userID}
+        )
+        imageURL = s3.read(profile['imageKey'])
```

Comment:
there'll be an error for this if keyholder is at default location. ^see above comment